core/src/main/java/com/sequenceiq/cloudbreak/reactor/api/event/cluster/dr/backup/DatabaseBackupSuccess.java

```diff
@@ -0,0 +1,17 @@
+package com.sequenceiq.cloudbreak.reactor.api.event.cluster.dr.backup;
+
+import com.sequenceiq.cloudbreak.reactor.api.event.cluster.dr.BackupRestoreEvent;
+
+import static com.sequenceiq.cloudbreak.core.flow2.cluster.datalake.dr.backup.DatabaseBackupEvent.DATABASE_BACKUP_FINISHED_EVENT;
+
+public class DatabaseBackupSuccess extends BackupRestoreEvent {
+
+    public DatabaseBackupSuccess(Long stackId) {
+        super(stackId, null, null);
+    }
+
+    @Override
+    public String selector() {
+        return DATABASE_BACKUP_FINISHED_EVENT.event();
```

Comment:
so we usually we do this the other way around. I would omit this part and let `EventSelectorUtil` take care of the selector here.
I would modify `DatabaseBackupEvent` like this:
`DATABASE_BACKUP_FINISHED_EVENT(EventSelectorUtil.selector(DatabaseBackupSuccess.class)),`
could you check the other events to be in line with this pattern?